README.md

```diff
@@ -0,0 +1,128 @@
+Firebase REST API. Eiffel Client
```

Comment:
General comment: try to add style to the Readme file, because at the moment is not easy to read.
Markdown syntax: https://daringfireball.net/projects/markdown/syntax
Online editors http://dillinger.io/, https://stackedit.io/